#### Science in general

1. Biesecker LG. (2013) Hypothesis-generating research and predictive medicine. Genome Res 23: 1051-1053.
2. Heard, Stephen B. "On whimsy, jokes, and beauty: can scientific writing be enjoyed?." Ideas in Ecology and Evolution 7.1 (2014).
3. Schwartz, Martin A. "The importance of stupidity in scientific research." Journal of Cell Science 121.11 (2008): 1771-1771.
4. Greenberg, Steven A. "How citation distortions create unfounded authority: analysis of a citation network." Bmj 339 (2009): b2680.

#### Sequencing

1. Sboner A, Mu XJ, Greenbaum D, Auerbach RK, Gerstein MB. (2011) The real cost of sequencing: Higher than you think! Genome Biol 12: 125-2011-12-8-125.
2. Hansen KD, Wu Z, Irizarry RA & Leek JT (2011) Sequencing technology does not eliminate biological variability. Nat Biotechnol 29(7): 572-573.

#### Microbiome (mostly 16S)

1. Goodrich, Julia K., et al. "Conducting a microbiome study." Cell 158.2 (2014): 250-262.
2. Brown J, de Vos WM, DiStefano PS, Dore J, Huttenhower C, et al. (2013) Translating the human microbiome. Nat Biotechnol 31: 304-308.
3. Hamady M, Knight R. (2009) Microbial community profiling for human microbiome projects: Tools, techniques, and challenges. Genome Research 19: 1141-1152.
4. Hugenholtz P, Tyson GW. (2008) Microbiology: Metagenomics. Nature 455: 481-483.
5. Knights D, Parfrey LW, Zaneveld J, Lozupone C, Knight R. (2011) Human-associated microbial signatures: Examining their predictive value. Cell Host Microbe 10: 292-296.
6. McMurdie PJ, Holmes S. (2014) Waste not, want not: Why rarefying microbiome data is inadmissible. PLoS Comput Biol 10: e1003531.
7.  Reeder J, Knight R. (2009) The 'rare biosphere': A reality check. Nat Methods 6: 636-637.
8. Wommack KE, Bhavsar J, Ravel J. (2008) Metagenomics: Read length matters. Appl Environ Microbiol 74: 1453-1463.
9. Wooley JC, Godzik A, Friedberg I. (2010) A primer on metagenomics. PLoS Comput Biol 6: e1000667.

#### RNAseq

1. Soneson C, Delorenzi M. (2013) A comparison of methods for differential expression analysis of RNA-seq data. BMC Bioinformatics 14: 91-2105-14-91.
2. Li, Jun, and Robert Tibshirani. "Finding consistent patterns: a nonparametric approach for identifying differential expression in RNA-Seq data." Statistical methods in medical research 22.5 (2013): 519-536.

#### Statistics (and data visualization)

1. Cumming G, Fidler F, Vaux DL. (2007) Error bars in experimental biology. J Cell Biol 177: 7-11.  
**_This has a lot of good example plots for SD, Variance, CI, etc._**
2. Nieuwenhuis S, Forstmann BU, Wagenmakers EJ. (2011) Erroneous analyses of interactions in neuroscience: A problem of significance. Nat Neurosci 14: 1105-1107.
3. Biesecker LG (2013) Hypothesis-generating research and predictive medicine. Genome Res 23(7): 1051-1053.
4. Knights D, Parfrey LW, Zaneveld J, Lozupone C & Knight R (2011) Human-associated microbial signatures: Examining their predictive value. Cell Host Microbe 10(4): 292-296.
5. Broadhurst D & Kell D (2006) Statistical strategies for avoiding false discoveries in metabolomics and related experiments. Metabolomics 2(4): 171-196.
6. Nakagawa S & Cuthill IC (2007) Effect size, confidence interval and statistical significance: A practical guide for biologists. Biol Rev Camb Philos Soc 82(4): 591-605.
7. Vaux DL, Fidler F & Cumming G (2012) Replicates and repeats—what is the difference and is it significant?. EMBO Rep 13(4): 291-296.
8. Altman, Douglas G., and J. Martin Bland. "Standard deviations and standard errors." Bmj 331.7521 (2005): 903.
9. Halsey, Lewis G., et al. "The fickle P value generates irreproducible results." nature methods 12.3 (2015): 179-185.
10. Blainey, Paul, Martin Krzywinski, and Naomi Altman. "Points of significance: replication." Nature methods 11.9 (2014): 879-880.
11. Weissgerber, Tracey L., et al. "Beyond bar and line graphs: time for a new data presentation paradigm." PLoS Biol 13.4 (2015): e1002128.

#### Bioinformatics/computational biology

1. Ciccarelli FD, Doerks T, von Mering C, Creevey CJ, Snel B, et al. (2006) Toward automatic reconstruction of a highly resolved tree of life. Science 311: 1283-1287.
1. Noble WS. (2009) A quick guide to organizing computational biology projects. PLoS Comput Biol 5: e1000424.

#### Misc

1. Hjortrup, Peter B., et al. "Gone fishing in a fluid trial." Critical care and resuscitation: journal of the Australasian Academy of Critical Care Medicine 18.1 (2016): 55-58.
1. Bennett, Craig M., et al. "Neural correlates of interspecies perspective taking in the post-mortem atlantic salmon: an argument for proper multiple comparisons correction." Journal of Serendipitous and Unexpected Results 1 (2011): 1-5.
1. Sender, Ron, Shai Fuchs, and Ron Milo. **Revised estimates for the number of human and bacteria cells in the body.** PLoS Biology (2016): DOI:10.1371/journal.pbio.1002533.

----------
### Blog posts
On contamination in low DNA abundance samples  
- http://blog.mothur.org/2014/11/12/TheKitome/  
- http://phenomena.nationalgeographic.com/2014/11/11/contaminomics-why-some-microbiome-studies-may-be-wrong/

-------
###Gloor Lab

[Gloor lab publications on Google Scholar](https://scholar.google.ca/citations?hl=en&user=909NxQEAAAAJ&view_op=list_works&sortby=pubdate)

**ALDEx2**

Fernandes, Andrew D., et al. "Unifying the analysis of high-throughput sequencing datasets: characterizing RNA-seq, 16S rRNA gene sequencing and selective growth experiments by compositional data analysis." Microbiome 2.1 (2014): 1.

**ALDEx**

Fernandes, Andrew D., et al. "ANOVA-like differential expression (ALDEx) analysis for mixed population RNA-Seq." PLoS One 8.7 (2013): e67019.

**metatranscriptome**

Macklaim, Jean M., et al. "Comparative meta-RNA-seq of the vaginal microbiota and differential expression by Lactobacillus iners in health and dysbiosis." Microbiome 1.1 (2013): 1.

**Compositional data analysis (CoDa) for sequencing**

Gloor, Gregory B., et al. "It’s all relative: analyzing microbiome data as compositions." Annals of Epidemiology (2016).


